data/curated/border-patrol/intro.md

```diff
@@ -0,0 +1,13 @@
+DESCRIPTION
+
+This week we're exploring [U.S. Customs and Border Protection (CBP) encounter data](https://www.cbp.gov/document/stats/nationwide-encounters):
+
+> Encounter data includes U.S. Border Patrol Title 8 apprehensions, Office of Field Operations Title 8 inadmissibles, and all Title 42 expulsions for fiscal years 2020 to date. Data is available for the Northern Land Border, Southwest Land Border, and Nationwide (i.e., air, land, and sea modes of transportation) encounters. Data is extracted from live CBP systems and data sources. Statistical information is subject to change due to corrections, systems changes, change in data definition, additional information, or encounters pending final review. Final statistics are available at the conclusion of each fiscal year.
+
+Thank you to [Tony Galván](https://www.linkedin.com/in/anthony-raul-galvan/) for curating this dataset and providing a [blog post](https://posit.cloud/content/9130042) that explores the data in more detail.
```

Comment:
For both here and meta.yaml: I can't view the content at this URL. I think this is your edit URL, not the published URL. I can't remember how to make content from posit.cloud public, but can help you sort it out if you need someone else to poke around! I recommend loading the URL in an incognito browser window (ctrl+shift+N) to see if someone who isn't logged in as you has access. I created #771 to make sure I check this every time, because I *almost* missed this!

Reply:
I changed access from private to "All Posit Cloud Users". I'm not sure there is a way to make it truly open to the "public"

Reply:
Doh! Still not available, even when logged in: 
> Content Unavailable
> Publishing on Posit Cloud has been deprecated and previously published data documents, applications, and APIs are no longer accessible.
> 
> RStudio and Jupyter code projects are unaffected by this change.
> 
> For more information, please see [our message about this change](https://docs.posit.co/cloud/whats_new/#sunsetting-posit-cloud-publishing).

Maybe link to one of the LinkedIn posts? It'd be awesome if you could migrate it to a public blog, but I understand that that can be a pretty big lift!

Reply:
(see above, but I'm keeping this comment unresolved so we remember to update both places)